### PR TITLE
Clarify the definition of a unit time step as 1 hour

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

**Comment by brianhelba** [comment]
@knappa Based on [a discussion](https://github.com/NutritionalLungImmunity/nlisim/pull/94/files/59778f8e8b37e3b87f6ea8bba900d1c4d7f59826#diff-484b66c810d03f13e21fb499f7a7dace292cf8488ffe4fc737e0e157b036d3e3) with @jbeezley, my understanding is that each time step (in particular, the `target_time` argument to `nlisim run __`) represents 1 hour.

Please let me know if this is correct, or should be interpreted differently. Regardless, the current documentation is clearly inconsistent.

**Review by jbeezley — APPROVED** [review]

**Comment by brianhelba** [comment]
@knappa Waiting for your approval on this too.

**Review by knappa — APPROVED** [review]
Yes, there is still some question about making sure that model parameters are correct for this timescale, but 'hours' are the correct unit.

**Pull request merged**
